extensions/linz/README.md

```diff
@@ -0,0 +1,66 @@
+# LINZ Top Level Extension Specification
+
+- **Title:** LINZ Top Level Extension
+- **Identifier:**
+  <https://linz.github.io/STAC/extensions/linz/json-schema/schema.json>
+- **Field Name Prefix:** NONE
+- **Scope:** Collection
+- **Extension
+  [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):**
+  Proposal
+- **Owner**: @your-gh-handles @person2
```

Comment:
We should probably specify LINZ as the owner, although they seem to want user IDs they can contact directly.

Reply:
Done. I added all of us